packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsummary.md

```diff
@@ -0,0 +1,5 @@
+<div class="common-image-format">
+
+![Sequence diagram showing Google enrollment summary](/img/authenticators/ios-authenticators-google-enrollment-summary.png)
+
+</div>
```

Comment:
Can you make the name of the steps in the diagram match those in the text or vice versa please?